pkg/controllers/managementuser/resourcequota/resource_quota_sync_test.go

```diff
@@ -4,14 +4,169 @@ import (
 	"reflect"
 	"testing"
 
-	v32 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
-
 	"github.com/stretchr/testify/assert"
 	corev1 "k8s.io/api/core/v1"
 	apiequality "k8s.io/apimachinery/pkg/api/equality"
 	"k8s.io/apimachinery/pkg/api/resource"
+
+	v32 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
 )
 
+func TestCompleteLimit(t *testing.T) {
+	type input struct {
+		newValues     *v32.ContainerResourceLimit
+		defaultValues *v32.ContainerResourceLimit
```

Comment:
Might be more readable like this:

```suggestion
		nsLimits     *v32.ContainerResourceLimit
		projectLimits *v32.ContainerResourceLimit
```

Reply:
Ok, Thanks!